docs/design.md

```diff
@@ -224,11 +260,11 @@ Normal
 TEAMMATES
 ```
 
-##Client
+## Client Component
 
-The Client component contains scripts that can connect to the application backend for things such as migrating data and calculating statistics.
+The Client component contains scripts that can connect to the application back end for administrative purposes, such as migrating data to a new schema and calculating statistics.
 
 Package overview:
 
-+ **`client.remoteapi`**: Classes needed to connect to the backend directly.
-+ **`client.scripts`**: Scripts that do things with the back end data.
+- **`client.remoteapi`**: Classes needed to connect to the back end directly.
+- **`client.scripts`**: Scripts that do things with the back end data.
```

Comment:
What do you think about `Scripts that reads/analyzes/manipulates the back end data`?